tests/e2e/main_test.go

```diff
@@ -277,3 +277,42 @@ func TestFailedRuleEvaluations(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGrafana(t *testing.T){
+	kClient := promClient.kubeClient
+	grafanaScale, err := kClient.AppsV1().Deployments("monitoring").GetScale(context.Background(), "grafana", metav1.GetOptions{})
```

Comment:
By getting the Deployment instead of the Scale object, you'll be able to get more info about the replicas, such as the number of ReadyReplicas for example.